pkg/event/dispatcher.go

```diff
@@ -41,13 +41,13 @@ type Dispatcher interface {
 
 // HTTPEventDispatcher is the HTTP implementation of the Dispatcher interface
 type HTTPEventDispatcher struct {
+	requester *utils.HTTPRequester
 }
 
 // DispatchEvent dispatches event with callback
-func (*HTTPEventDispatcher) DispatchEvent(event LogEvent) (bool, error) {
+func (eventDispacher *HTTPEventDispatcher) DispatchEvent(event LogEvent) (bool, error) {
```

Comment:
nit: should this be shortened to something like `ed` like we do for other func receivers?

Reply:
addressed